src/collection/views/CollectionEdit.tsx

```diff
@@ -55,13 +55,16 @@ import {
 
 interface CollectionEditProps extends RouteComponentProps {}
 
+let currentCollection: Avo.Collection.Collection | undefined;
+let setCurrentCollection: (collection: Avo.Collection.Collection) => void;
```

Comment:
@bertyhell Again, dit was de oorzaak. 😭

Reply:
Wat gaat dit juist oplossen want heb dit al vaker tegengekomen? Dit lijkt me gwn niet de way-to-go 😅